### PR TITLE
Add instructions

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/Viva/issues/601

### CODE REVIEW

File: Viva/insights/advanced/admin/upload-org-data-first.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ After the data successfully validates and processes, the overall data-upload tas
 
 To upload your .csv file, follow these steps:
 
+1. In the Microsoft admin center, go to the [setup tab](https://admin.microsoft.com/adminportal/home#/featureexplorer) and select **Microsoft Viva**, then **Viva Insights**. You'll need to enter your credentials if you're not already signed in.
+1. Under **Add-on Plan**, select **Manage privacy, data, and manager settings in Advanced Insights**.
+1. Select **Organizational data** from the Menu.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/admin/upload-org-data-first.md:46]:
```suggestion
1. Select **Organizational data** from the menu.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@lilyolason 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by lilyolason** [comment]
Hi @scanum -- I was editing this document and made a change that should close this issue. I've provided the direct URL, rather than using the MAC.

**Pull request closed without merging**